homeassistant/components/homewizard/sensor.py

```diff
@@ -342,21 +350,6 @@ class HomeWizardSensorEntityDescription(
         device_class=SensorDeviceClass.POWER,
         value_fn=lambda data: data.monthly_power_peak_w,
     ),
-    HomeWizardSensorEntityDescription(
-        key="total_gas_m3",
-        name="Total gas",
-        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
-        device_class=SensorDeviceClass.GAS,
-        state_class=SensorStateClass.TOTAL_INCREASING,
-        value_fn=lambda data: data.total_gas_m3,
-    ),
-    HomeWizardSensorEntityDescription(
-        key="gas_unique_id",
-        name="Gas meter identifier",
-        icon="mdi:alphabetical-variant",
-        entity_category=EntityCategory.DIAGNOSTIC,
-        value_fn=lambda data: data.gas_unique_id,
-    ),
```

Comment:
This sensor is now added as 'name' for the device. Maybe we want to do something extra as well?